### ISSUE TITLE
Calendar support with Markwhen or Almanac

### ISSUE THREAD

**Issue by kaafir786** [gsantner/markor#2330]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **enhancement/feature request**. Not a [bug or question](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The wanted feature/enhancement is not present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Marder**. Don't worry; Markor and Marder appear as completely separate applications. You can install both side-by-side, and Markor's settings are not touched. If your desired feature is present, you don't need to open this issue. The change will be part of the next Markor update.)

### Description

Basically supporting plain text Calendar using either [Markwhen](https://markwhen.com/) or [Almanac](https://github.com/ParthTri/Almanac)

Markwhen is more powerful and Almanac is more inline of current project (It will be easier to implement it). Go with either and thanks for managing my life.

### Information

Android version: 2.12.2
Device: Nokia C21 Plus
App Version: 11

### Source

F-Droid

### Format / File type

Something else (-> Additional info)

**Comment by harshad1** [comment]
We have a todo.txt format. What minimal features would be needed to make todo.txt serve as a calendar

**Comment by kaafir786** [comment]
*1. Adding a separate tab like quicknote and quicktodo
2. Being able to set start and end time
3. Grey out event list which is passed
4. Supporting paragraphs not just a line
_Elaborating last point_
I mean in almanac you can write a line with date and in below lines keep Setting time and event name
*5. Preview these events in calendar (web based)
*6. Reminders for next event

*These things also need to be implemented with Almanac.

I will still prefer going with Almanac as it also have a program for pc.

**Comment by gsantner** [comment]
Never heard of it humbly said but if  want it supported, feel free to 
implement Almanac or find somebody who does it. At same time, take also 
in account that you are supposed to help maintaining it after it gets added.
Markor has many formats supported by now, but little amount of people 
maintaining each. And it's not "implementing it once and it's done forever".

I don't know other plaintext software supporting that however, which is 
bad for something as big as calendars (which you would expect to show up 
at least in a real calendar GUI/app).

Can just recommend to you what also was said above: Check if maybe 
todo.txt (due date) or orgmode fits your flow. They are also supported 
in other software, i.e. todo.txt for Thunderbird :smiley: .

The only thing I can offer you to do on my own for this format is to 
register a file extension, if it not uses .txt but .almanac or alike.

**Comment by kaafir786** [comment]
> Never heard of it humbly said but if 
Sarcasm😔?, I know actually my mental health is not good.

Currently I am not a programmer but will work on it after I learn programming (1.5 more yeasr probably)

For now me can close it.
Thanks for support sir. ☺️

**Comment by kaafir786** [comment]
Probably I behave like boss on all open source projects.
I fit nowhere.

(Offtopic)
